src/sql/platform/telemetry/common/telemetryKeys.ts

```diff
@@ -97,13 +97,14 @@ export const enum TelemetryAction {
 	RunQuery = 'RunQuery',
 	RunQueryStatement = 'RunQueryStatement',
 	RunQueryString = 'RunQueryString',
+	SearchCompleted = 'SearchCompleted',
+	SearchStarted = 'SearchStarted',
 	ShowChart = 'ShowChart',
 	StopAgentJob = 'StopAgentJob',
+	ToggleOnActualQueryExecutionPlan = 'ToggleOnActualQueryExecutionPlan',
```

Comment:
This is supposed to be generic for execution plans right? It should be `ToggleOnActualExecutionPlan` in that case